<ui version="4.0" >
 <class>AddAuto</class>
 <widget class="QMainWindow" name="AddAuto" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>564</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>564</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>AutoSelect</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="systray.qrc" >
    <normaloff>:/pics/active-icon-0.xpm</normaloff>:/pics/active-icon-0.xpm</iconset>
  </property>
  <property name="dockNestingEnabled" >
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>564</width>
     <height>355</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>564</width>
     <height>355</height>
    </size>
   </property>
   <widget class="QWidget" native="1" name="widget" >
    <property name="geometry" >
     <rect>
      <x>5</x>
      <y>5</y>
      <width>546</width>
      <height>355</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize" >
     <size>
      <width>546</width>
      <height>355</height>
     </size>
    </property>
    <property name="maximumSize" >
     <size>
      <width>546</width>
      <height>355</height>
     </size>
    </property>
    <widget class="QFrame" name="frame" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>321</height>
      </rect>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="Role" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>160</y>
        <width>260</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="Desktop" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>290</y>
        <width>400</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="Class" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>120</y>
        <width>260</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Name" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>260</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_class" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All classes</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_desktop" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>270</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All desktops</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_name" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All names</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_host" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All hosts</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_role" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>160</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All Roles</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Host" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>260</width>
        <height>22</height>
       </rect>
      </property>
      <property name="whatsThis" >
       <string comment="This hase the host " >The host name.</string>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Title" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>220</y>
        <width>400</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_title" >
      <property name="geometry" >
       <rect>
        <x>290</x>
        <y>200</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>All titles</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel_title" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>200</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Title</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="projectName" >
      <property name="geometry" >
       <rect>
        <x>9</x>
        <y>9</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>Project A - Milestone one - Bug 32</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel_desktop" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>270</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Desktop</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget" >
     <property name="geometry" >
      <rect>
       <x>440</x>
       <y>10</y>
       <width>101</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1" >
      <item>
       <widget class="QPushButton" name="buttonOk" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel" >
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Grab" >
        <property name="text" >
         <string>Grab</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="Spacer1" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>180</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonHelp" >
        <property name="text" >
         <string>&amp;Help</string>
        </property>
        <property name="shortcut" >
         <string>F1</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget" >
   <property name="features" >
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QTableView" name="view" />
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="systray.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AddAuto</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>498</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>416</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_host</sender>
   <signal>clicked(bool)</signal>
   <receiver>Host</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel" >
     <x>280</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_name</sender>
   <signal>clicked(bool)</signal>
   <receiver>Name</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>314</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel" >
     <x>268</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_class</sender>
   <signal>clicked(bool)</signal>
   <receiver>Class</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>307</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel" >
     <x>269</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_role</sender>
   <signal>clicked(bool)</signal>
   <receiver>Role</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>311</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_title</sender>
   <signal>clicked(bool)</signal>
   <receiver>Title</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>312</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_desktop</sender>
   <signal>clicked(bool)</signal>
   <receiver>Desktop</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>308</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
